src/js/components/Notification/Notification.js

```diff
@@ -173,6 +173,7 @@ const Notification = ({
       pad={undefined}
       direction="row"
       gap="small"
+      {...rest}
```

Comment:
Adding this will address https://github.com/grommet/grommet/issues/6134
```suggestion
      id={toast ? undefined : id}
      {...rest}
```